zowe-rest-api-sample-spring/build.gradle

```diff
@@ -42,7 +43,7 @@ def commonsProject = { ->
         try {
             return project(':zowe-rest-api-commons-spring')
         } catch (ignored) {
-            return 'org.zowe:zowe-rest-api-commons-spring:0.2.1'
+            return 'org.zowe:zowe-rest-api-commons-spring:0.2.0-SNAPSHOT'
```

Comment:
I'm curious on this change, should the minor have been bumped here?

Reply:
Thank you for noticing. I need to figure out how to explain it and how to make it less confusing.

Reply:
The version in the repository should be always `0.0.0-SNAPSHOT` since the real published version is taken by from the tag during publishing. I tried to describe it in https://github.com/zowe/sample-spring-boot-api-service/pull/41/files#diff-f5758d353c2bb60f873830856c7ce6e0 and https://github.com/zowe/sample-spring-boot-api-service/pull/41/files#diff-314794aef9dd7518d7dd053490329145. In the API Mediation Layer repo (zowe/api-layer), we use https://github.com/researchgate/gradle-release. This was not easy to do in CircleCI since CircleCI can only read from the GitHub repository. The releasing is base on GitHub tags, when a new tags in format `vm.n.p` is created, new version is published with the specified version number. Some version needs to be used for local builds so `0.0.0-SNAPSHOT` is used.